sooch/commands/help.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+import discord
+import sooch.message as msg
+
+
+# Using a separate file for now, could merge with misc later (if needed)
+
+async def help(client: discord.Client,
+               message: discord.Message,
+               content: list[str]) -> Optional[discord.Embed]:
+
+    # Initialize the embed and text (the value for the field)
+    embed = discord.Embed()
+    help_text = ""
+
+    # Checks if there are any arguments
+    if not content[1:]:
+        for command in msg.commands:
+            help_text += f"`{command}` *sor* `{''.join(msg.commands.get(command).aliases)}` \n"
```

Comment:
Did you mean `or` instead of `sor`? The code also doesn't seem to work with multiple aliases atm either